extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/SimpleScheduler.java

```diff
@@ -141,6 +141,20 @@ public void pause() {
         }
     }
 
+    @Override
+    public void pause(String identity) {
+        if (identity == null) {
+            LOGGER.warn("Cannot pause - identity is null");
+            return;
+        }
+        for (ScheduledTask task : scheduledTasks) {
+            if (identity.equals(task.trigger.id)) {
```

Comment:
I also think this should use the https://github.com/quarkusio/quarkus/blob/c573fbee41915278e71475f5bf9ed276aaf26b42/extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/util/SchedulerUtils.java#L69 as the identity can also be an expression.

Reply:
I don't think that it's necessary - the value of `task.trigger` is already parsed...

Reply:
I am talking about the parameter. A user can still give a raw `identity`, right?

Reply:
All usages of `identity` parameter are now using lookUpPropertyValue()

Reply:
I'm not sure I'm following. I don't see a point in passing a property expression such as `${my.config.property}`... you can just obtain the config property and pass the value directly.

Reply:
I agree with you that someone can find the property corresponding to the expression and pass it but as a user having I'd expect the scheduler to handle this for me since I can set my identity as an expression https://github.com/quarkusio/quarkus/blob/ca6056f240546d7c0597b57b7bae6e9a87be1749/extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/Scheduled.java#L46-L47 having an ability an use the same expression to pause and resume a scheduled method is way handy rather than me having to parse the expression wherever I need to call pause/resume.

Reply:
Ok, I'm +0 on this. But we should probably add a test?

Reply:
+1 on having a test for this. @adrianfiedler would you be able to add a test for this?

Reply:
Awesome @adrianfiedler great work, thanks!